bodhi/__init__.py

```diff
@@ -78,6 +75,10 @@ def get_buildinfo(request):
     return defaultdict(dict)
 
 
+def get_releases(request):
+    import bodhi.services.releases
+    return bodhi.services.releases.query_releases_html(request)['releases']
```

Comment:
Doesn't this mean every time any page is rendered, it has to go through the `releases` cornice `Service`, the `ListReleaseSchema`, 3 different validators, and then a db query? Seems a bit heavyweight. Can't we do a simple `@classmethod`+ class-level-cache hack like we do with `Release.get_tags`? https://github.com/fedora-infra/bodhi/blob/develop/bodhi/models/models.py#L316-L330